.github/workflows/release-latest-binaries.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+  push:
+    branches:
+    # Push events on master branch
+    - master
+
+name: Create Latest Binaries Release
+
+jobs:
+  build-and-release:
+    name: Build and Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    # Taken from https://github.com/actions/checkout#push-a-commit-using-the-built-in-token
+    - name: Update latest tag
+      run: |
+        git config user.name 'github-actions'
+        git config user.email 'github-actions@github.com'
+        git tag -f -a 'latest' -m 'Latest binaries release'
+        git push -f --tags
+
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.14'
+    - run: go version
+
+    - name: Build Go binaries
+      shell: bash
+      run: ./build/build-binaries.sh -v 'latest'
+
+    # From https://github.com/github/hub#github-actions
+    - name: Create GitHub Release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      shell: bash
+      run: |
+        set -x
+
+        # Install the latest version of hub
+        ./build/install-hub.sh
+
+        # If the release exists already, delete it so it can be recreated
+        # For some reason, `hub edit` fails to update assets and crashes with various network errors
+        if hub release | grep -q 'latest'; then
+          bin/hub release delete 'latest'
+        fi
+
+        # Create the release with all binaries
+        # Set the pre-release flag to indicate that it is not ready for production and may be unstable
+        bin/hub release create $(printf -- ' --attach=%s' ./*-amd64) -m 'Binaries latest release' 'latest' --prerelease
```

Comment:
let's rename release to build to emphasize it's not a real stable release.

Reply:
Done.